deploy/environments/applications.yaml

```diff
@@ -119,7 +119,6 @@ spack:
     - psp-validation%gcc ^neuron%intel
     - py-atlalign ^py-six@1.15.0
     - py-atlas-building-tools
-    - py-atldld
```

Comment:
why remove this?

Reply:
I did not intend to, maybe a missclick sorry. Should I recreate the PR ?

Reply:
Just re-add these lines / rebase properly on top of latest `develop`